package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "ckeditor5",
+  "name": "@ckeditor/ckeditor5",
```

Comment:
Why don't we use a name like `ckeditor5-repository` or a similar one to clearly describe what this name represents? The `@ckeditor/ckeditor5` name is too ambiguous.

Reply:
I went for `ckeditor5-root`